src/components/Charts.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const Charts = () => {
+    return (
+        <>
+            <h1>Charts</h1>
+            <p>Charts to go here below</p>
+            <Link to="/">
+                <button>Home</button>
+            </Link>
+        </>
```

Comment:
Since this is a component, rather a page, it would be best to wrap this with a `section` or a `div`. One big reason is semantic HTML and also because if you would wanna style it, you'll need to give it a class and right now there's no possibility of that. Avoid using `<></>` too much, specially like this, because it's a misuse of it. it's only for partials or smaller blocks of repeating code mostly.

Reply:
done